src/server/multi_test.cc

```diff
@@ -511,9 +556,9 @@ TEST_F(MultiTest, MultiOOO) {
   // OOO works in LOCK_AHEAD mode.
   int mode = absl::GetFlag(FLAGS_multi_exec_mode);
   if (mode == Transaction::LOCK_AHEAD || mode == Transaction::NON_ATOMIC)
-    EXPECT_EQ(200, metrics.coordinator_stats.tx_type_cnt[ServerState::OOO]);
+    EXPECT_EQ(200, metrics.coordinator_stats.tx_shard_ooo_cnt);
```

Comment:
what's the semantics of tx_shard_ooo_cnt? 
in fact, I do not think they belong to ServerState now.

Reply:
Total number of ooo runs on all shards, the testcase has a single shard multi command, so it's the same as the number of commands

Reply:
Excluding QUICK runs, or should we include them too?